tools/changelog/Makefile

```diff
@@ -0,0 +1,32 @@
+OS ?= $(shell uname -s | tr '[[:upper:]]' '[[:lower:]]')
+ARCH ?= $(shell uname -m)
+TOOL_NAME ?= changelog
+BUILD_DIR ?= build/$(OS)/$(ARCH)/
+VERSION ?= v0.0.0-dev
+
+lint:
+	@golangci-lint run ./... --out-format colored-line-number
+
+test:
+	@gotestsum --format github-actions ./... -- -shuffle on -timeout 2m -race
+
+binary:
+	@echo "Building for $(OS)/$(ARCH) and writing to $(BUILD_DIR)"
+	@mkdir -p "$(BUILD_DIR)"
+	@GOOS=$(OS) GOARCH=$(ARCH) go build -o "$(BUILD_DIR)/" -ldflags="-s -w"
+
+tarball: TARBALL_NAME = "$(TOOL_NAME)-$(VERSION)-$(OS)-$(ARCH).tar.gz"
```

Comment:
Don't use quotes when setting make variables - they are not special and just end up literally in the value. They will be placed on the command line literally and you will end up unquoted as it is expanded inside quotes already (the quotes in the var will close the quotes in the command).